cloudsmith/datadog_checks/cloudsmith/check.py

```diff
@@ -0,0 +1,235 @@
+from json import JSONDecodeError
+from urllib.error import HTTPError
+
+import requests
+from requests.exceptions import InvalidURL, Timeout
+
+from datadog_checks.base import AgentCheck, ConfigurationError
+
+METRIC = "/metrics/entitlements/"
+QUOTA = "/quota/"
+TIMEOUT = 10
+WARNING_QUOTA = 75
+CRITICAL_QUOTA = 85
+
+
+class CloudsmithCheck(AgentCheck):
+    def __init__(self, name, init_config, instances):
+        super(CloudsmithCheck, self).__init__(name, init_config, instances)
+
+        self.base_url = self.instance.get('url')
+        self.api_key = self.instance.get('cloudsmith_api_key')
+        self.org = self.instance.get('organization')
+
+        self.log.debug('Cloudsmith monitoring starting on %s', self.base_url)
+        self.tags = []
+
+        self.tags.append('BASE_URL:{}'.format(self.base_url))
+        self.tags.append('CLOUDSMITH_ORG:{}'.format(self.org))
+
+    def get_full_path(self, path):
+        url = self.base_url + path + self.org
+        return url
+
+    # Get stats from REST API as json
+    def get_api_json(self, url):
+        self.tags.append('FULL_URL:{}'.format(url))
+
+        try:
+            key = self.api_key
+            headers = {"X-Api-Key": key, "content-type": "application/json"}
+            req = requests.get(url, timeout=TIMEOUT, headers=headers)
```

Comment:
```suggestion
            req = self.http.get(url, extra_headers=headers)
```
Do not use requests directly